packages/lesswrong/server/postgresFunctions.ts

```diff
@@ -1,248 +1,305 @@
 import { allUserProfileFields } from "./userProfileUpdates";
 
+export type PostgresFunction = {
+  source: string,
+  overload?: string,
+  dependencies?: SchemaDependency[],
+}
```

Comment:
Changes in this file are mainly just indentation to make room for the new fields